doc/languages-frameworks/coq.section.md

```diff
@@ -5,9 +5,11 @@
 The Coq derivation is overridable through the `coq.override overrides`, where overrides is an attribute set which contains the arguments to override. We recommend overriding either of the following
 
 * `version` (optional, defaults to the latest version of Coq selected for nixpkgs, see `pkgs/top-level/coq-packages` to witness this choice), which follows the conventions explained in the `coqPackages` section below,
-* `customOCamlPackage` (optional, defaults to `null`, which lets Coq choose a version automatically), which can be set to any of the ocaml packages attribute of `ocaml-ng` (such as `ocaml-ng.ocamlPackages_4_10` which is the default for Coq 8.11 for example).
+* `customOCamlPackages` (optional, defaults to `null`, which lets Coq choose a version automatically), which can be set to any of the ocaml packages attribute of `ocaml-ng` (such as `ocaml-ng.ocamlPackages_4_10` which is the default for Coq 8.11 for example).
 * `coq-version` (optional, defaults to the short version e.g. "8.10"), is a version number of the form "x.y" that indicates which Coq's version build behavior to mimic when using a source which is not a release. E.g. `coq.override { version = "d370a9d1328a4e1cdb9d02ee032f605a9d94ec7a"; coq-version = "8.10"; }`.
 
+The associated package set can be optained using `mkCoqPackages coq`, where `coq` is the `coq` derivation to use.
```

Comment:
I understand what you mean, but it reads weird otherwise. I think this still conveys the message.
```suggestion
The associated package set can be optained using `mkCoqPackages coq`, where `coq` is the derivation to use.
```

Reply:
Yeah, I feel like maybe the second coq should be normal font, or maybe capitalized. Something like : where `coq` is the Coq derivation to use. Would that be better ?

Reply:
I'm not sure which edit you suggest more, but after looking at both a bit more, I think the first one you suggested is clearer.